lib/stripe/request.ex

```diff
@@ -215,8 +215,22 @@ defmodule Stripe.Request do
     params =
       Enum.reduce(to_cast, params, fn key, params ->
         case params[key] do
-          %{__struct__: _, id: id} -> put_in(params[key], id)
-          _ -> params
+          %{__struct__: _, id: id} ->
+            put_in(params[key], id)
+
+          list when is_list(list) ->
+            new_list =
+              Enum.map(list, fn %{id: id} = list_item ->
+                case id do
+                  %{__struct__: _, id: id} -> put_in(list_item.id, id)
+                  _ -> list_item
+                end
+              end)
+
+            put_in(params[key], new_list)
+
```

Comment:
Can a test be added to illuminate this necessary `is_list` case?

Reply:
Of course, will add them later today.

Reply:
@snewcomer I've had some less time than I thought, sorry for delay. Also, could you tell me how exactly would you like to see this tested? Where tests should go? Thanks!

Reply:
@siepet 👋 Lmk if you want some help with this!

Reply:
@snewcomer If you could simply point me to the place where should I place tests for this!